plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -302,7 +302,7 @@ class GeneralServiceFormSection extends Component {
           Services
         </h2>
         <p>
-          Configure your service below. Start by giving your service a name.
+          Configure your service below. Start by giving your service an Id.
```

Comment:
It should be All Caps - http://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized